motor/providers/microsoft/platform.go

```diff
@@ -34,7 +34,40 @@ func (p *Provider) Roles() map[string]struct{} {
 	return p.rolesMap
 }
 
+func getTitleForPlatformName(name string) string {
+	switch name {
+	case "azure-compute-vm":
+		return "Azure Virtual Machine"
```

Comment:
This is different from https://github.com/mondoohq/cnquery/pull/963/files#diff-914af7db2b557306fddc6f2c387e9de21ddd9ad0868906028516c2d8d25fa76cR14. Not sure whether this is relevant.